tests/e2e/test_pvc_creation_performance.py

```diff
@@ -44,3 +47,40 @@ def test_pvc_creation_measurement_performance(self):
                 f"PVC creation time is {create_time} and greater than 1 second"
             )
         logging.info("PVC creation took less than a 1 second")
+
+    @pytest.mark.usefixtures(base_setup.__name__)
+    @polarion_id('OCS-1620')
+    @bugzilla('1741612')
+    def test_multiple_pvc_creation_measurement_performance(self):
+        """
+        Measuring PVC creation time of 120 PVCs in 60 seconds
+        """
+        number_of_pvcs = 120
+        log.info('Start creating new 120 PVCs')
+
+        pvc_objs = create_multiple_pvcs(
+            sc_name=self.sc_obj.name,
+            namespace=defaults.ROOK_CLUSTER_NAMESPACE,
+            number_of_pvc=number_of_pvcs,
+            size=self.pvc_size,
+            wait=False
+        )
+        for pvc_obj in pvc_objs:
+            wait_for_resource_state(pvc_obj, constants.STATUS_BOUND)
+        start_time = measure_pvc_creation_time(
+            'CephBlockPool', pvc_objs[0].name, return_start_time=True
+        )
+        end_time = measure_pvc_creation_time(
```

Comment:
Do you want to aggregate the total time taken by all PVC's instead? There is a possibility that last PVC was created > 60 seconds?

Reply:
If I understand you correctly so not really

Reply:
Resolving this based on JIRA ticket